contrib/node/src/python/pants/contrib/node/tasks/node_repl.py

```diff
@@ -70,3 +70,10 @@ def launch_repl(self, targets):
 
         repl_session = node_repl.run()
         repl_session.wait()
+    # Repl task doesn't make any sence for Node.js with yarnpkg for the following reasons:
```

Comment:
Would you mind opening a github issue with this information, and then leaving a shorter TODO/comment here that links to it?